API/src/microservices/Assessment-Service/index.js

```diff
@@ -23,6 +23,22 @@ exports.submit = (assessment) => {
 
       resolve(assesment);
 
+    } catch (err) {
+      reject();
+    }
+  });
+};
+
+exports.retrieve = () => {
+  return new Promise(async (resolve, reject) => { //eslint-disable-line
+    try {
+      let assesmentList = await new Assessments().where({deleted_at: null}).fetchAll().catch(function (e) {
+        console.log('error in retrieving assessments')
+        resolve(assesmentList.toJSON());
```

Comment:
```suggestion
        resolve(assessmentList.toJSON());
```
few typos here.  I think vscode has some spellchecker extensions you can download if interested.

Reply:
Corrected my typos.